### ISSUE TITLE
Fix test breakages on Xcode 10 Final Release.

### ISSUE THREAD

**Issue by tirodkar** [google/EarlGrey#751]

**Comment by hassankaz** [comment]
Is there any update for this issue? I can't run the tests on xcode Version 10.0 (10A255) at all. Is there any workaround?  I appreciate any help.  Thanks!

**Comment by tirodkar** [comment]
What is the build failure that you're seeing? Is it just related to scripts not being found?

**Comment by hassankaz** [comment]
![image](https://user-images.githubusercontent.com/1616330/46170005-73753200-c251-11e8-92fb-c13c3dbe647f.png)
 this is the error i'm seeing with a test run

**Comment by hassankaz** [comment]
Yes so it can't find earlgrey scripts.  Any update on this issue or workaround? thank you so much!

**Comment by tomfclearscore** [comment]
We have seen this issue too.  Is anybody working on it?

**Comment by tirodkar** [comment]
@hassankaz That doesn't look like a Build failure. That's a legitimate EarlGrey test failing. Do you have an animation or so on the main thread that is taking longer than 30 seconds to complete?

**Comment by tirodkar** [comment]
@tomfclearscore The one workaround we know of is to disable the new build system that comes with Xcode 10. TTBOMK, if you just rerun the command though, it should work fine.
